bindings/go/src/fdb/fdb_test.go

```diff
@@ -127,37 +135,46 @@ func TestEstimatedRangeSize(t *testing.T) {
 func TestReadTransactionOptions(t *testing.T) {
 	fdb.MustAPIVersion(API_VERSION)
 	db := fdb.MustOpenDefault()
-	_, err := db.ReadTransact(func(rtr fdb.ReadTransaction) (interface{}, error) {
+	_, txClose, err := db.ReadTransact(context.Background(), func(rtr fdb.ReadTransaction) (interface{}, error) {
```

Comment:
Shouldn't we call in the method body `defer .Close()` on the returned future from the `rtr.Get` method? Otherwise we have a text example that introduces a memory leak?